components/common/buttons/ButtonClose.tsx

```diff
@@ -0,0 +1,16 @@
+import type { FC } from 'react';
+
+import CloseIcon from '@mui/icons-material/Close';
+import IconButton, { type IconButtonProps } from '@mui/material/IconButton';
+
+const ButtonCloseComponent: FC<IconButtonProps> = props => {
+  const { 'aria-label': ariaLabel, size, ...restProps } = props;
+
+  return (
+    <IconButton {...restProps} aria-label="delete">
+      <CloseIcon aria-label={ariaLabel || 'close'} fontSize={size || 'medium'} />
```

Comment:
It's unclear why `ariaLabel` passed to `ButtonCloseComponent` is used for `CloseIcon`. From a developer's perspective, I expect that that prop will be used for `IconButton`.

Reply:
'+